i18n/locales/source/blockly-mooc/common.json

```diff
@@ -1730,7 +1730,6 @@
   "projects": "Projects",
   "projectsSubHeading": "{project_count} projects created",
   "projectsSubHeadingMillions": "Over {project_count} million projects created",
-  "print": "Print",
```

Comment:
I don't know why this is deleted. It still exists in the repo (`apps/i18n/common/en_us.json`)

Reply:
It appeared twice in here. hmm. OK THEN.